### ISSUE TITLE
Chat in Editor should have same background as chat in panel

### ISSUE THREAD

**Issue by jo-oikawa** [microsoft/vscode#261493]
Expected: When I move/open a chat to the editor, it should appear the same and use the same colors.
Actual: It's using a lighter (in dark mode) color from the editor. 

Since we haven't designed the chat elements for this background color, it's having some unintended consequences. Examples:
- https://github.com/microsoft/vscode/issues/234866
- Todo styling looks incomplete/broken
- Color contrast ratios for foreground/background (ex. checkpoints UI) **does not meet accessibility standards**
